src/components/TransactionItemRow/index.tsx

```diff
@@ -69,6 +71,9 @@ function TransactionItemRow({
         }
     }, [hovered, isSelected, styles.activeComponentBG, styles.hoveredComponentBG]);
 
+    const merchantName = getMerchant(transactionItem);
+    const isMerchantEmpty = merchantName === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT;
```

Comment:
Lets use the isPartialMerchant from TransactionUtils
```suggestion
    const isMerchantEmpty = isPartialMerchant(merchantName);
```